.docksal/commands/release

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+## Make a new release and deploys it.
+##
+## Usage: fin release
+
+# Abort if anything fails
+set -e
+
+# Backup the environments.
+echo -e "Backup Dev"
+fin terminus backup:create ${hostingsite}.dev --keep-for=365
+
+# Check Test enviroment exists
+# wc = word count. -l prints the number of lines.
+# tr = translating or deleting characters. -s replaces repeated characters.
+var_test_count=$(fin terminus env:list --field=id ${hostingsite} | grep 'test' | wc -l | tr -s " ");
+if [[ "$var_test_count" -eq "0" ]]; then
+  # Backup Test.
+  echo -e "Backup Test"
+  fin terminus backup:create ${hostingsite}.test --keep-for=365
+fi
+
+# Check Live enviroment exists
+var_live_count=$(fin terminus env:list --field=id ${hostingsite} | grep 'live' | wc -l | tr -s " ");
+if [[ "$var_live_count" -eq "0" ]]; then
+  # Backup Test.
+  echo -e "Backup Test"
+  fin terminus backup:create ${hostingsite}.live --keep-for=365
+fi
+
+# Create a new tag.
+var_release_previous=$(gh release view)
+if [ -z "$var_release_previous" ]; then
+    var_tag_new=0.0.1
+else
+    var_tag_new=$(git describe --tags --abbrev=0 | awk -F. '{OFS="."; $NF+=1; print $0}')
+fi
+
+echo -e "Creating a new tag."
+git tag -a $var_tag_new -m "Autogenerated deployment release tag"
+
+echo -e "Push new tag."
+git push origin $var_tag_new
+
```

Comment:
`gh release create`
If a matching git tag does not yet exist, one will automatically get created from the latest state of the default branch.

Reply:
We can probably move this all to GitHub which would be safer than local.

Reply:
ok, that works!